File: angular/src/app/model/coordonnee.ts
export class Coordonnee {
  constructor (private _telFixe?: string, private _telPortable?: string, private _mail?: string){

  }

  get telFixe(): string {
    return this._telFixe;
  }

  set telFixe(value: string) {
    this._telFixe = value;
  }

  get telPortable(): string {
    return this._telPortable;
  }

  set telPortable(value: string) {
    this._telPortable = value;
  }

  get mail(): string {
    return this._mail;
  }

  set mail(value: string) {
    this._mail = value;
  }
}

File: java/src/main/java/com/sopra/Factory/model/Coordonnee.java
package com.sopra.Factory.model;

import javax.persistence.Column;
import javax.persistence.Embeddable;

import com.fasterxml.jackson.annotation.JsonView;
import com.sopra.Factory.model.view.JsonViews;

@Embeddable
public class Coordonnee {
	
	@JsonView(JsonViews.Common.class)
	@Column(name = "tel_fixe")
	private String telFixe;
	
	@Column(name = "tel_portable")
	@JsonView(JsonViews.Common.class)
	private String telPortable;
	
	@Column(name = "email")
	@JsonView(JsonViews.Common.class)
	private String mail;

	public Coordonnee() {
		super();
	}

	public String getTelFixe() {
		return telFixe;
	}

	public void setTelFixe(String telFixe) {
		this.telFixe = telFixe;
	}

	public String getTelPortable() {
		return telPortable;
	}

	public void setTelPortable(String telPortable) {
		this.telPortable = telPortable;
	}

	public String getMail() {
		return mail;
	}

	public void setMail(String mail) {
		this.mail = mail;
	}

	
	
}
